joomla/cassiopeia
# Change danger and breadcrumb active item colors

Pull Request for Issue #113 and #114.

### Summary of Changes
Changed color values for danger and breadcrumb active item

### Testing Instructions
Run npm ci or complie scss
Go to author login and check the contract of the error text when nothing entered
Check the contrast of the active item on breadcrumbs

### Expected result
![grafik](https://user-images.githubusercontent.com/9153168/94566997-4bb76b00-026b-11eb-9140-192d5f8f581a.png)

![grafik](https://user-images.githubusercontent.com/9153168/94567116-65f14900-026b-11eb-9c1c-e491f87bd5bb.png)

The contrast is WCAG conform

### Actual result
The contrast is not WCAG conform
